snappy/snapp_test.go

```diff
@@ -240,15 +240,15 @@ func (s *SnapTestSuite) TestUbuntuStoreRepositoryUpdates(c *C) {
 	var err error
 	s.storeCfg.BulkURI, err = url.Parse(mockServer.URL + "/updates/")
 	c.Assert(err, IsNil)
-	snap := store.NewUbuntuStoreSnapRepository(s.storeCfg, "")
```

Comment:
Woah, thanks for doing this. Make me wonder how this ended up being called snap in the first place. Oh well!